[2024-03-01T15:09:22.373+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-03-01T15:09:22.398+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2024-02-29T00:00:00+00:00 [queued]>
[2024-03-01T15:09:22.403+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-01T15:09:22.414+0000] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-02-29 00:00:00+00:00
[2024-03-01T15:09:22.420+0000] {standard_task_runner.py:60} INFO - Started process 8958 to run task
[2024-03-01T15:09:22.423+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v01', 'get_name', 'scheduled__2024-02-29T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpp__mrp8w']
[2024-03-01T15:09:22.427+0000] {standard_task_runner.py:88} INFO - Job 27: Subtask get_name
[2024-03-01T15:09:23.343+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_taskflow_api_v01.get_name scheduled__2024-02-29T00:00:00+00:00 [running]> on host 417e643aed36
[2024-03-01T15:09:23.406+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='dn' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v01' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-02-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-29T00:00:00+00:00'
[2024-03-01T15:09:23.408+0000] {python.py:202} INFO - Done. Returned value was: Jerry
[2024-03-01T15:09:23.431+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v01, task_id=get_name, execution_date=20240229T000000, start_date=20240301T150922, end_date=20240301T150923
[2024-03-01T15:09:23.504+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T15:09:23.528+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
